docs/fundamentals/flow/when-things-go-wrong.md

```diff
@@ -113,29 +110,29 @@ YAML `with` block.
 
 If the retry policy is exhausted for a given request, the error is reported back to the corresponding client.
 
-The resulting error message will contain the *network address* of the failing Executor.
-If multiple replicas are present, all addresses will be reported - unless the Flow is deployed using Kubernetes, in which
-case the replicas are managed by k8s and only a single address is available.
+The resulting error message contains the *network address* of the failing Executor.
+If multiple replicas are present, all addresses are reported - unless the Flow is deployed using Kubernetes, in which
+case the replicas are managed by Kubernetes and only a single address is available.
 
-Depending on the client-to-gateway protocol, and they type of error, the error message will be returned in one of the following ways:
+Depending on the client-to-gateway protocol, and the type of error, the error message is returned in one of the following ways:
 
 **Could not connect to Executor:**
 
-- **gRPC**: A response with the gRPC status code 14 (*UNAVAILABLE*) is issued, and the error message is contained in the `details` field
-- **HTTP**: A response with the HTTP status code 503 (*SERVICE_UNAVAILABLE*) is issued, and the error message is contained in `response['header']['status']['description']`
-- **WebSocket**: The stream closes with close code 1011 (*INTERNAL_ERROR*) and the message is contained in the WS close message
+- **gRPC**: A response with the gRPC status code 14 (*UNAVAILABLE*) is issued, and the error message is contained in the `details` field.
+- **HTTP**: A response with the HTTP status code 503 (*SERVICE_UNAVAILABLE*) is issued, and the error message is contained in `response['header']['status']['description']`.
+- **WebSockets**: The stream closes with close code 1011 (*INTERNAL_ERROR*) and the message is contained in the WebSockets close message.
```

Comment:
Are we sure it should be plural? our `protocol` variable accepts the `singular` term

Reply:
From [Wikipedia](https://en.wikipedia.org/wiki/WebSocket) it refers to both "the WebSocket protocol" (I'm paraphrasing) or "WebSockets".

I'm fine either way

Reply:
judge if it can be missleading on having `websocket` singular accepted in the code.

Reply:
Edited as appropriate, based on what I saw from googling